lib/check_graphite/projection.rb

```diff
@@ -0,0 +1,39 @@
+require 'attime'
+require 'bigdecimal'
+require 'nagios_check'
+require 'linear-regression'
+
+module CheckGraphite
+  module Projection
+    def self.included(base)
+      base.on '--projection FUTURE_TIMEFRAME', :default => '2days' do |timeframe|
+        options.send('processor=', method(:projected_value))
+        options.send('timeframe=', timeframe)
```

Comment:
The Ruby-ish thing to do is to use `:processor=` and `:timeframe=` (i.e. symbols not strings when referring to methods).